backend/app/services/repository.py

```
@@ -66,42 +66,23 @@ async def get_repositories(
-        except Exception as e:
-            self.logger.error(f"Error fetching repositories: {e}")
-            raise HTTPException(
-                status_code=502,
-                detail=f"Error fetching repositories: {str(e)}"
-            )
-        
+            try:
+                provider = self.providers[provider_type]
+                repos = await provider.get_repositories(user, page=1, limit=100)
+                all_repos.extend(repos)
+            except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)